src/components/filmDescription/FilmDescription.js

```diff
@@ -14,6 +14,7 @@ const FilmDescription = ({ film }) => {
           <h2 className="film-description__header">{title}</h2>
           <strong className="film-description__useful-info display-block">{tagline}</strong>
           <span>
+
             Year:
```

Comment:
Why do you need mixed test with data and spaces here. May be it is a good idea to use string template here?